virt/monitoring/virt-running-cluster-checkups.adoc

```diff
@@ -12,6 +12,8 @@ xref:../../virt/monitoring/virt-running-cluster-checkups.adoc#virt-measuring-lat
 Verifies network connectivity and measures latency between two virtual machines (VMs) that are attached to a secondary network interface.
 xref:../../virt/monitoring/virt-running-cluster-checkups.adoc#virt-checking-cluster-dpdk-readiness_virt-running-cluster-checkups[DPDK checkup]::
 Verifies that a node can run a VM with a Data Plane Development Kit (DPDK) workload with zero packet loss.
+xref:../../virt/monitoring/virt-running-cluster-checkups.adoc#virt-checking-storage-configuration_virt-running-cluster-checkups[Storage checkup]::
```

Comment:
Should links be in an additional resources section? At first I thought the heading were blue as part of AsciiDocs syntax for defintions :smile: .